### DISCUSSION TITLE
how to create the pickle file for a custom dataset?

### DISCUSSION THREAD

**Discussion by gynaa** [open-mmlab/mmdetection3d#2283] [Q&A]
Hello everyone,

I want to train some models on a custom dataset. So I have to create a pickle file with the data. I looked at the kitti pickle file and it has a few parameters, that I am not sure where to get them from: num_points_in_gt, gt_idx and group_id. 

Could someone help me out here please?

**Comment by JingweiZhang12** [comment]
`num_points_in_gt, gt_idx and group_id.` is the annotation of specific datasets.  If you don't have these annotations, you can ignore it.
